themes/bootstrap3/templates/search/searchbox.phtml

```diff
@@ -99,7 +100,14 @@
     <?php endif; ?>
     <button type="submit" class="btn btn-primary"><?=$this->icon('search') ?> <?=$this->transEsc("Find")?></button>
     <?php if ($advSearch): ?>
-      <a href="<?=$this->url($advSearch) . (($this->searchId ?? false) ? '?edit=' . $this->escapeHtmlAttr($this->searchId) : $hiddenFilterParams) ?>" class="btn btn-link" rel="nofollow"><?=$this->transEsc("Advanced")?></a>
+      <?php
+        if ($results) {
```

Comment:
This could be condensed significantly if you just use a ternary to set the 'query' rather than duplicating the whole line of code...

```
$advSearchQuery = $results ? ['edit' => $results->getSearchId()] : $hiddenFilters;
$advSearchLink = $this->url($advSearch, [], ['query' => $advSearchQuery]);
```

Reply:
Thanks, done!